### PR TITLE
DEV-317: Edit postbiuld.py to produce zip file for the Mac Launcher.

### PR DESCRIPTION
**Pull request by danteruiz**
Ticket - https://highfidelity.atlassian.net/browse/DEV-317

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20584/HighFidelity-Beta-PR15987-58dea69.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20584/HighFidelity-Beta-Interface-PR15987-58dea69.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20584/HighFidelity-Beta-PR15987-58dea69.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20584/HighFidelity-Beta-Interface-PR15987-58dea69.dmg

File: tools/ci-scripts/postbuild.py [hunk 1]
```diff
@@ -148,9 +148,30 @@ def signBuild(executablePath):
...
+
+    cpackCommand = [
+        'cpack',
+        '-G', 'ZIP',
+        '-D', "CPACK_PACKAGE_FILE_NAME={}".format(archiveName),
+        '-D', "CPACK_INCLUDE_TOPLEVEL_DIRECTORY=OFF"
+    ]
+    print("Create ZIP version of installer archive")
+    print(cpackCommand)
+    hifi_utils.executeSubprocess(cpackCommand, folder=launcherBuildPath)
```
Comment (huffman) [inline tools/ci-scripts/postbuild.py]: What happens if this fails? Does the script exit?

**Comment by danteruiz** [comment]
This is the zip file the Jenkins produced - https://ci.highfidelity.io/job/pr-build/20584/label=macos/s3/download/HQ%20Launcher-PR15987-58dea69-mac.zip

File: tools/ci-scripts/postbuild.py (hunk 1 above)
Comment (danteruiz) [reply tools/ci-scripts/postbuild.py]: Not sure what exactly happens if it fails.

Comment (danteruiz) [reply tools/ci-scripts/postbuild.py]: That call is used through out our build scripts

Comment (huffman) [reply tools/ci-scripts/postbuild.py]: I just checked - it raises an exception

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20584/HighFidelity-Beta-PR15987-58dea69.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20584/HighFidelity-Quest-Beta-PR15987-58dea69.apk)

**Pull request merged**